src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  width: 1200px;
```

Comment:
Avoid fixing the container size unless required. Let the content size dictate the width to prevent overflow or accidental scroll bars.